src/Softeq.NetKit.Chat.Domain/TransportModels/Response/Localization/LocalizationResponse.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Developed by Softeq Development Corporation
+// http://www.softeq.com
+
+using System.Collections;
+
+namespace Softeq.NetKit.Chat.Domain.TransportModels.Response.Localization
+{
+    public class LocalizationResponse
+    {
+        public string Key { get; set; }
```

Comment:
Are we returning default system message text in english? can not find

Reply:
yes, we are getting the key from the appsettings.json and sending it using SendSystemMessageAsync